src/store/modules/webhooks/mutations.ts

```diff
@@ -0,0 +1,10 @@
+import { MutationTree } from "vuex";
+import WebhooksState from "./WebhooksState";
+
+const mutations: MutationTree<WebhooksState> = {
+  setWebhooks (state, payload: any) {
```

Comment:
Define mutation types and then use the types, take reference from job module

Reply:
updated